client/src/styles/NrMap.css

```diff
@@ -0,0 +1,27 @@
+.card {
```

Comment:
please avoid .css files and use Styled Components instead (see other existing code for reference)